src/decays/decay_functions.cpp

```diff
@@ -0,0 +1,337 @@
+// ====================================================================
+// This file is part of FlexibleSUSY.
+//
+// FlexibleSUSY is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published
+// by the Free Software Foundation, either version 3 of the License,
+// or (at your option) any later version.
+//
+// FlexibleSUSY is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with FlexibleSUSY.  If not, see
+// <http://www.gnu.org/licenses/>.
+// ====================================================================
+
+#include <cmath>
+#include <limits>
+
+#include "decay_functions.hpp"
+#include "dilog.hpp"
+#include "trilog.hpp"
+#include "Li4.hpp"
+#include "numerics2.hpp"
+#include "wrappers.hpp"
+
+namespace flexiblesusy {
+
+namespace {
+
+using namespace std::literals::complex_literals;
+
+/// Eq.(2.31) of hep-ph/0503172
+double RT_general(double x) noexcept
+{
+   const std::complex<double> z(x, 0.0);
+
+   return std::real(
+      3.*(1. - 8.*z + 20.*z*z)/std::sqrt(4.*z - 1.)*std::acos((3.*z - 1.)/(2.*std::pow(z, 3./2.)))
+      - (1. - z)/(2.*z)*(2. - 13.*z + 47.*z*z)
+      - 3./2. * (1. - 6.*z + 4.*z*z)*std::log(z)
+   );
+}
+
+/// Eq.(2.7) of hep-ph/0503173
+double calc_A(double b) noexcept
+{
+   const double log_b {std::log(b)};
+   const double log_ratio {std::log((1 + b) / (1 - b))};
+
+   return (1 + b * b) *
+             (4. * dilog((1 - b) / (1 + b)) + 2. * dilog(-(1 - b) / (1 + b)) -
+              3. * log_ratio * std::log(2 / (1 + b)) -
+              2. * log_ratio * log_b) -
+          3. * b * std::log(4. / (1 - b * b)) - 4. * b * log_b;
+}
+
+} // anonymous namespace
```

Comment:
I love anonymous namespaces! :)
They keep the code local to the translation unit and tell the compiler that no external linkage is required, so it can inline the code etc.
So, I'd keep it.

Reply:
As above I am outvoted :).